test/gtest/ucp/test_ucp_am.cc

```diff
@@ -1848,7 +1849,14 @@ class test_ucp_am_nbx_rndv_memtype : public test_ucp_am_nbx_rndv {
 
 UCS_TEST_P(test_ucp_am_nbx_rndv_memtype, rndv)
 {
-    test_am_send_recv_memtype(64 * UCS_KBYTE);
+    const auto *cfg                 = &sender().worker()->context->config.ext;
+    const size_t rndv_frag_size     = cfg->rndv_frag_size[UCS_MEMORY_TYPE_HOST];
```

Comment:
minor:
```suggestion
    const auto &cfg                 = sender().worker()->context->config.ext;
    const size_t rndv_frag_size     = cfg.rndv_frag_size[UCS_MEMORY_TYPE_HOST];
```

Reply:
i'd keep it as is to avoid restarting CI if you don't mind

Reply:
> i'd keep it as is to avoid restarting CI if you don't mind

ok